cmd/controller-manager/app/options/options.go

```diff
@@ -50,7 +50,7 @@ const (
 	defaultLeaderElectionNamespace      = "kube-system"
 )
 
-var defaultOSBAPIPreferredVersion = osb.Version2_12().HeaderValue()
+var defaultOSBAPIPreferredVersion = osb.LatestAPIVersion().HeaderValue()
```

Comment:
going along with all the other flag defaults. would be const, but can't have a const result of a function call.